server/api/region.go

```diff
@@ -140,3 +141,115 @@ func (h *regionsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 	h.rd.JSON(w, http.StatusOK, regionsInfo)
 }
+
+const defaultRegionLimit = 16
+
+func (h *regionsHandler) GetTopWriteFlow(w http.ResponseWriter, r *http.Request) {
```

Comment:
can we combine TopRead/WriteFlow? They use nearly same codes.

Reply:
Addressed. PTAL @siddontang